src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -0,0 +1,2463 @@
+            Tuple of `torch.FloatTensor` (one for each layer) of shape `(batch_size, num_heads, sequence_length,
+            sequence_length)`. Attentions weights of the decoder's cross-attention layer, after the attention softmax,
+            used to compute the weighted average in the cross-attention heads.
+        intermediate_hidden_states (`torch.FloatTensor` of shape `(config.decoder_layers, batch_size, num_queries, hidden_size)`, *optional*, returned when `config.auxiliary_loss=True`):
+            Intermediate decoder activations, i.e. the output of each decoder layer, each of them gone through a
+            layernorm.
+    """
+
+    intermediate_hidden_states: Optional[torch.FloatTensor] = None
+
+
+@dataclass
+class MaskFormerPixelLevelModuleOutput(ModelOutput):
+    """MaskFormer's pixel level module output. It returns both the last and (optionally) the hidden states from the `encoder`
+    and `decoder`. By default, the `encoder` is a Swin Transformer and the `decoder` is a Feature Pyramid Network
+    (FPN).
+
+        The `encoder_last_hidden_state` are referred on the paper as **images features**, while
+        `decoder_last_hidden_state` as **pixel embeddings**
+         Args:
+            encoder_last_hidden_state (`torch.FloatTensor` of shape`(batch_size, num_channels, height, width)`):
+                Last hidden states (final feature map) of the last stage of the encoder.
+            encoder_hidden_states (`tuple(torch.FloatTensor)`, *optional*, returned when `output_hidden_states=True` is passed or when `config.output_hidden_states=True`):
+                Tuple of `torch.FloatTensor` (one for the output of the embeddings + one for the output of each stage)
+                of shape `(batch_size, num_channels, height, width)`. Hidden-states (also called feature maps) of the
+                model at the output of each stage.
+            decoder_last_hidden_state (`torch.FloatTensor` of shape`(batch_size, num_channels, height, width)`):
+                Last hidden states (final feature map) of the last stage of the decoder.
+            decoder_hidden_states (`tuple(torch.FloatTensor)`, *optional*, returned when `output_hidden_states=True` is passed or when `config.output_hidden_states=True`):
+                Tuple of `torch.FloatTensor` (one for the output of the embeddings + one for the output of each stage)
+                of shape `(batch_size, num_channels, height, width)`. Hidden-states (also called feature maps) of the
+                model at the output of each stage.
+    """
+
+    encoder_last_hidden_state: Optional[torch.FloatTensor] = None
+    decoder_last_hidden_state: Optional[torch.FloatTensor] = None
+    encoder_hidden_states: Optional[Tuple[torch.FloatTensor]] = None
+    decoder_hidden_states: Optional[Tuple[torch.FloatTensor]] = None
+
+
+class MaskFormerPixelDecoderOutput(BaseModelOutput):
```

Comment:
```suggestion
class MaskFormerPixelDecoderOutput(ModelOutput):
```
Similarly, please inherit from `ModelOutput` and copy the attributes.